app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.setdefault(self.name, self)
```

Comment:
You don't need setdefault method here, because you don't need to return value if it already exist